contracts/ERC721A.sol

```diff
@@ -276,7 +280,7 @@ contract ERC721A is Context, ERC165, IERC721A {
         bytes memory _data
     ) public virtual override {
         _transfer(from, to, tokenId);
-        if (to.isContract()) if(!_checkContractOnERC721Received(from, to, tokenId, _data)) {
+        if (to.code.length != 0) if(!_checkContractOnERC721Received(from, to, tokenId, _data)) {
```

Comment:
Is checking code.length equivalent to using the `EXTCODESIZE` opcode?

Reply:
yes